src/utils/HTMLUtils.js

```diff
@@ -1,17 +1,31 @@
 const cheerio = require("cheerio");
 
+const { replace } = require("lodash/fp");
+
 const isPlainText = elem => typeof elem === "string" && !elem.startsWith("<");
 
 const getInnerHTML = elem => (isPlainText(elem) ? elem : cheerio(elem).html());
 
+const unescapePiping = value =>
```

Comment:
Could this be pushed down into `innerHTML`?

Reply:
I guess it could, I was going for the whole 1 function does 1 thing approach, hence why I introduced the `getInnerHTMLWithPiping` function. But I could push the unescaping down into `getInnerHTML` if that's clearer?

Reply:
Yeah, I'm not sure what the solution is. This is kinda workaround because it is being escaped by cheerio. So it feels like its more of it's concern but I can see your point.

Leave it but I think this should be re-visited at some point to see if we can drop cheerio. AFAICT the only reason we are using it is to pull out those different bits of guidance?

Reply:
We can certainly look again at our use of cheerio. I guess some of the need for cheerio would go away if piping wasn't represented as a HTML `span` tag, but rather an entity in its own right.

I don't know what an alternative to cheerio would look like though. We still need to deal with markup entered by a user somehow and escaping user-entered HTML does seem like a sensible thing to do in general.

Reply:
My assumption is that if we were providing guidance as multiple UI fields we could treat it all as a string and just do a replace on the things that look like piping.

So there would be no need to use cheerio at all. Publisher wouldn't care that it is HTML.